### ISSUE TITLE
Document how to run lombok when its path contains spaces

### ISSUE THREAD

**Issue by Taximan** [redhat-developer/vscode-java#135]
As the title says, I am having this issue on a gradle project that was created in InteliJ (and worked just fine there with the lombok plugin). I do get normal intelisense but after adding the vmargs option the language server won't boot up.

##### Environment
- Operating System: W10
- JDK version: 8
- Visual Studio Code version: 1.8.1
- Java extension version:  0.0.8

Following https://github.com/redhat-developer/vscode-java/wiki/Lombok-support this is my settings.json

```json
// Place your settings in this file to overwrite the default settings
{
    "editor.insertSpaces": true,
    "editor.tabSize": 2,
    // C:\Program Files\Java
    "java.jdt.ls.vmargs": "-javaagent:C:\\Program Files\\Java\\lombok.jar -Xbootclasspath/a:C:\\Program Files\\Java\\lombok.jar"
}
```
I imagine this could a problem with

a:) the fact that gradle support is recent and its not a supported feature
b:) my path format is bad (in this case I recommend adding some example for windows to the wiki page for clarity)

It's probably option b. As a java newcommer I have no idea how the formats should look like and querying google is hard since I don't really know what to really ask for however basing myself off http://stackoverflow.com/questions/22310414/how-to-configure-lombok-in-eclipse-luna I think I kinda got right or did I?

**Comment by Taximan** [comment]
Oh god, got around it by moving my lombok file to a different place (I assume the issue to be with "Program Files"). Now I have 

```json
   "java.jdt.ls.vmargs": "-javaagent:C:\\lombok.jar -Xbootclasspath/a:C:\\lombok.jar"
```

Can be closed.

**Comment by fbricon** [comment]
looks like you'd need to escape the paths containing whitespaces. Probably like:

```
"java.jdt.ls.vmargs": "-javaagent:\"C:\\Program Files\\Java\\lombok.jar\" -Xbootclasspath/a:\"C:\\Program Files\\Java\\lombok.jar\""
```

I haven't checked, but let me know if it works so we can update the wiki page.

**Comment by Taximan** [comment]
Nope, same error.

**Comment by fbricon** [comment]
Ok, I'll look into it next week.

**Comment by fbricon** [comment]
I found a problem in the VM Args parsing. I should be able to provide a fix soon.

**Comment by fbricon** [comment]
Fixed with 80f4443.

Instructions updated in https://github.com/redhat-developer/vscode-java/wiki/Lombok-support
